src/loaders/genome_collection/compute_genome_taxa_count.py

```diff
@@ -0,0 +1,173 @@
+import argparse
+import json
+import os
+from collections import defaultdict
+
+import src.common.storage.collection_and_field_names as names
+import src.loaders.common.loader_common_names as loader_common_names
+from src.common.gtdb_lineage import (
+    parse_gtdb_lineage_string,
+    GTDB_RANK_ABBREV_TO_FULL_NAME,
+)
+from src.common.hash import md5_string
+from src.loaders.common.loader_helper import convert_to_json
+from src.service.data_products import taxa_count
+
+"""
+PROTOTYPE - Prepare genome taxa count data and identical ranks in JSON format for arango import.
+
+usage: compute_genome_taxa_count.py [-h] --load_ver LOAD_VER [--kbase_collection KBASE_COLLECTION] [-o OUTPUT]
+                                    [--input_source {GTDB,genome_attributes}]
+                                    load_files [load_files ...]
+
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  load_files            Files containing genome taxonomy (e.g. ar53_taxonomy_r207.tsv, computed_genome_attribs.json)
+  --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1).
+
+optional arguments:
+  --kbase_collection KBASE_COLLECTION
+                        KBase collection identifier name (default: GTDB).
+  -o OUTPUT, --output OUTPUT
+                        Output JSON file path.
+  --input_source {GTDB,genome_attributes}
```

Comment:
Is there a reason to continue supporting GTDB direct input? You should be able to get the same results by running the script that takes the metadata file and produces what would normally be produced from the calculation parsing script, and then running this script on that, right?

Reply:
See comment chain below - we can't do this until the GTDB metadata parser and the compute -> parse scripts use the same key for gtdb lineage. Made a ticket for this.